lesson01.calculator/index.html

```diff
@@ -1,22 +1,318 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title>Web Calculator</title>
-    <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <style>
-        body {
+  <title>Web Calculator</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+    body {
+      font-family: arial, sans-serif;
+    }
 
-        }
-    </style>
+    .calc {
+      display: grid;
+      grid: repeat(6, 1fr) / repeat(4, 1fr);
+      grid-gap: 5px;
+      box-sizing: border-box;
+      padding: 5px;
+      height: 95vh;
+      grid-template-areas:
+        "display display display display"
+        "clear   clear   del     divide"
+        "seven   eight   nine    multiply"
+        "four    five    six     minus"
+        "one     two     three   plus"
+        "sign    zero    point   result";
+      border: 1px solid black;
+      background-color: lightyellow;
+    }
+
+    button {
+      background-color: lightblue;
+      font-size: 9vmin;
+      font-weight: bold;
+    }
+
+    button:hover {
+      background-color: lightcyan;
+    }
+
+    button.one {
+      grid-area: one;
+    }
+
+    button.two {
+      grid-area: two;
+    }
+
+    button.three {
+      grid-area: three;
+    }
+
+    button.four {
+      grid-area: four;
+    }
+
+    button.five {
+      grid-area: five;
+    }
+
+    button.six {
+      grid-area: six;
+    }
+
+    button.seven {
+      grid-area: seven;
+    }
+
+    button.eight {
+      grid-area: eight;
+    }
+
+    button.nine {
+      grid-area: nine;
+    }
+
+    button.zero {
+      grid-area: zero;
+    }
+
+    button.point {
+      grid-area: point;
+    }
+
+    button.sign {
+      grid-area: sign;
+    }
+
+    button.divide {
+      grid-area: divide;
+    }
+
+    button.multiply {
+      grid-area: multiply;
+    }
+
+    button.minus {
+      grid-area: minus;
+    }
+
+    button.plus {
+      grid-area: plus;
+    }
+
+    button.result {
+      grid-area: result;
+    }
+
+    button.clear {
+      grid-area: clear;
+    }
+
+    button.del {
+      grid-area: del;
+    }
```

Comment:
Think about a way to use `grid` without so much css (Try to remove lines 41-75 and see what happen)